src/ucp/wireup/wireup_cm.c

```diff
@@ -423,16 +423,17 @@ static ucs_status_t ucp_cm_ep_init_lanes(ucp_ep_h ep,
 
 static unsigned ucp_cm_client_uct_connect_progress(void *arg)
 {
-    ucp_ep_h ep                       = arg;
-    ucp_worker_h worker               = ep->worker;
-    ucp_context_h context             = worker->context;
-    ucp_wireup_ep_t *cm_wireup_ep     = ucp_ep_get_cm_wireup_ep(ep);
-    void *ucp_addr                    = NULL; /* Set to NULL to call ucs_free
-                                                 safely */
-    unsigned ep_init_flags            = 0;
-    unsigned ep_init_flags_variants[] = {
-        UCP_EP_INIT_KA_ALLOW_AUX_TL_RSC, 0, UCP_EP_INIT_CREATE_AM_LANE_ONLY
+    static const unsigned ep_init_flags_variants[] = {
```

Comment:
IMO should be called  "ep_init_flags_prio" because it's shorter, and we it means it's order of priority for connection establishment

Reply:
done